### PR TITLE
Support circle obstacles and path display

### PR DESCRIPTION
**Pull request by ecourtois**
Add -n/--no-wait option to allows to start monitoring the robot without waiting for the initialization sequence.
Add -r/--remote option to specify a remote device providing the serial port connected to the robot.
Add circle obstacles support.
Update rectangle obstacles support.
Integrate dyn_obstacles in RobotState.
Support and display expected path.

### CODE REVIEW

File: cogip/entities/dynobstacle.py [hunk 1]
```diff
@@ -53,3 +53,46 @@ def set_size(self, length: int, width: int) -> None:
         """
         self.mesh.setXExtent(width)
         self.mesh.setYExtent(length)
+
+
+class DynRoundObstacleEntity(Qt3DCore.QEntity):
```
Comment (gdoffe) [inline cogip/entities/dynobstacle.py]: Maybe could we keep names aligned with mcu-firmware and replacing 'Round' by 'Circle' ?

File: docs/usage/simulator.md [hunk 2]
```diff
@@ -42,14 +45,23 @@ It can be changed using:
...
 The default serial port can be modified using the environment variable `DEFAULT_UART`.
 
 To specify an other serial port, use the `-D/--device <UART_DEVICE>` CLI option.
 
+## Remote Mode
+
+If a robot is connected to a serial port on a remote device, use `-r/--remote <REMOTE>`
+to specify this device.
+
+The argument must identify a device that can connected using `ssh` without password.
```
Comment (gdoffe) [inline docs/usage/simulator.md:61]: that **is** connected

File: cogip/models/models.py [hunk 3]
```diff
@@ -206,6 +182,31 @@ def __hash__(self):
         return hash((type(self),) + tuple(self.__root__))
 
 
+class RobotState(BaseModel):
+    """
+    This contains information about robot state,
+    like mode, cycle, positions and speed.
```
Comment (gdoffe) [inline cogip/models/models.py:188]: and obstacles

**Review by gdoffe — CHANGES_REQUESTED** [review]
https://github.com/cogip/simulation/pull/13/commits/63ce6f32ee2c6bc7a6ee0e3c4dabc82b2e49f933:
'The connection to the remote device **uses** ssh.'

https://github.com/cogip/simulation/pull/13/commits/766069ec87c1e6f236f7c2eaed147a7b2ebaf1e0: Why ? not obvious. ;)

https://github.com/cogip/simulation/pull/13/commits/adeea179706f846ed45aa89afc6da2b2dafe2dd4: Why ? Is it the full path ? an intermediate one ? What is this path in the simulator context ?

https://github.com/cogip/simulation/pull/13/commits/3622230a83c0557bcfe7063d17ca6a782d2bdfd6 and https://github.com/cogip/simulation/pull/13/commits/6c5fab94e0fb9c9a3afc420a21f351de46d77312: remove 'simple', title will be shorter and it is not simple to me ! xD

File: cogip/entities/dynobstacle.py (hunk 1 above)
Comment (gdoffe) [reply cogip/entities/dynobstacle.py]: Nevermind, I saw your further commit.

File: cogip/models/models.py (hunk 3 above)
Comment (ecourtois) [reply cogip/models/models.py]: Done

**Review by gdoffe — APPROVED** [review]
LGTM !

**Pull request merged**